src/components/regionalization/RegionalizationButton/RegionalizationButton.tsx

```diff
@@ -24,9 +24,9 @@ export default function RegionalizationButton({
         iconPosition="left"
         onClick={() => setIsRegionalizationModalOpen(true)}
       >
-        {content ? (
+        {postalCode ? (
           <>
-            <span>{content}</span>
+            <span>{postalCode}</span>
           </>
         ) : (
           <span>Set your location</span>
```

Comment:
Remove the fragment, ins't necessary. Also, add key on both span.

Reply:
Good call on the unneeded fragment! I wonder if merging the 2 `span`s into 1 like the following would also work?

```jsx
<span>{ postalCode ?? 'Set your location' }</span>
```

Reply:
Thanks for the suggestion, @filipewl. It worked.

Reply:
@filipewl It worked, but I think it's best to keep the spans separated in order to have different keys. What do you think?